plugins/sso-auth/lib/zssHandler.js

```diff
@@ -24,7 +24,7 @@ class ZssHandler {
     this.logger = context.logger;
     this.instanceID = serverConf.instanceID;
     this.sessionExpirationMS = DEFAULT_EXPIRATION_MS; //ahead of time assumption of unconfigurable zss session length
-    const zoweInstanceId = process.env['ZOWE_INSTANCE'];
+    const zoweInstanceId = process.env['ZWE_zowe_cookieIdentifier'];
```

Comment:
please do serverConf.cookieIdentifier
and export ZWED_cookieIdentifier=ZWE_zowe_cookieIdentifier in convert-env in zlux-app-server.
Basically I want to see 0 occurrences of process.env in zlux-server-framework, ideally. It makes it much harder to track down issues like the one we see here. This is a WIP, we'll have even better cleanup later.

Reply:
resolved